src/pages/rechner/frage.tsx

```diff
@@ -0,0 +1,60 @@
+import { NextPage } from 'next';
+import { Page } from '../../layouts/page';
+import React, { useEffect, useState } from 'react';
+import { InputField } from '../../elements/input-field';
+import { Copy } from '../../identity/copy';
+import { Label } from '../../identity/label';
+
+type Props = {
+  previousImpact: string;
+  initialAnswer: string;
+};
+
+const IMPACT_AIRPLANE_HOUR_KG_CO2 = 0.29064;
+
+const Frage: NextPage<Props> = ({ previousImpact = '', initialAnswer = '' }) => {
+  const [answer, setAnswer] = useState(
+    typeof window !== 'undefined' ? window.localStorage.getItem('answer') || initialAnswer : ''
+  );
+  //ich glaub da müssti eigentlich epis anders im local storage denn mache => und de impact ganz am schluss hinzuefüege
+  const [impact, setImpact] = useState(
+    typeof window !== 'undefined' ? window.localStorage.getItem('impact') || previousImpact : ''
+  );
+
+  useEffect(() => {
+    setImpact((parseInt(answer) * IMPACT_AIRPLANE_HOUR_KG_CO2).toString());
+    window.localStorage.setItem('answer', answer);
+    window.localStorage.setItem('impact', impact);
```

Comment:
Good idea to start working with local storage so you have something to persist your data without already needing an api or something 👍 
Something to think about in the context of a real api would be that it won't be practical to send every keystroke the user makes to the api like you do now with the local storage. You probably should work with the react state to handle the user input and choose a later action to persist that state to the local storage / api like when the user switches to the next question.